### ISSUE TITLE
Explicit dark mode for navigation bar

### ISSUE THREAD

**Issue by ig9000** [dozingcat/Vector-Pinball#79]
Unfortunately, the beautiful esthetic of this game is slightly compromised on a Samsung Galaxy S10 or similar due to the UI defaulting to a white navigation bar for apps which don't specify otherwise (even if so-called global dark mode is enabled). In dark environments it's actually quite distracting, so if it can be explicitly set to dark in the app (some apps seem to do this) it would be an enhancement. Alternately, if the app could run in fullscreen mode with the extra area at screen bottom active as flipper buttons, that might be even better. Regardless, thank you for maintaining this game!

![screenshot_20210217-141310_vector_pinball](https://user-images.githubusercontent.com/75760264/108326885-9e90b100-717f-11eb-9786-37cd3af8dbd4.png)

**Comment by dozingcat** [comment]
Wow, that is annoying, and seems to be common on many devices (e.g. https://stackoverflow.com/questions/51187710/android-navigationbarcolor-doesnt-change-to-black-only-to-black). It doesn't happen on any of my devices, so if you are able to build the app and install it on your phone, can you try replacing `app/src/main/res/values-21/styles.xml` with this?

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="Theme.VPTheme" parent="android:Theme.Material.NoActionBar.Fullscreen">
        <item name="android:navigationBarColor">#010101</item>
    </style>
</resources>
```

**Comment by ig9000** [comment]
Unfortunately, not set up for building APKs (it's on the TODO list somewhwere). The interesting bug you reference might be related, but in general what I've noticed is that newer material apps seem to get the navigation bar right, where older ones seem to leave it white regardless of dark/light theme chosen in the app. One interesting exception is [ImapNote2](https://github.com/N0ury/ImapNote2/tree/master/ImapNote2/src/main/res), which is old and unmaintained, but which does produce a black navigation bar despite (because of?) not even providing a choice of theme. Most other apps predating use of the navbar gui element get a white one. No idea why.

An example of a modern app successfully defining a black navbar (by referencing a named color) would be [Book Reader](https://gitlab.com/axet/android-book-reader/-/blob/HEAD/app/src/main/res/values/styles.xml).

If I manage to get gradle set up in the near future, I'll try to be more helpful with the troubleshooting.

**Comment by dozingcat** [comment]
I was able to borrow an LG phone that has a similar white navigation bar. Setting `navigationBarColor` in styles.xml didn't have any effect, but calling [Window.setNavigationBarColor](https://developer.android.com/reference/android/view/Window#setNavigationBarColor(int)) sets the color as expected. I'll add both of these fixes and hopefully at least one will work on most devices.

**Comment by ig9000** [comment]
Nice, feel free to close this issue if you're confident in the fix, and thanks for looking into it!

**Comment by dozingcat** [comment]
The fixes are in the latest 1.9.5 release. Please let me know if you're still seeing this behavior.

**Comment by ig9000** [comment]
Confirming that the issue is fixed for me in the 1.9.6 release from f-droid. Thanks again, I may reference this fix in tickets for some other apps that might benefit from it.
